asteroid/masknn/attention.py

```diff
@@ -0,0 +1,229 @@
+import torch.nn.functional as F
+import torch.nn as nn
+from torch.nn.modules.activation import MultiheadAttention
+from asteroid.masknn import activations, norms
+import torch
+from asteroid.utils import has_arg
+
+class ImprovedTransformedLayer(nn.Module):
```

Comment:
I know they call it like that but is this really a transformer layer in your opinion?

Reply:
it is MHA on top of standard DPRNN layer